### DISCUSSION TITLE
Diffusion With Offset Noise

### DISCUSSION THREAD

**Discussion by Njbx** [lllyasviel/ControlNet#32] [Ideas]
An interesting article that explains why Diffusion Contrast is so bad and offers a solution.

https://www.crosslabs.org/blog/diffusion-with-offset-noise

> So it seems like not only does SD not have the ability to produce overly dark or light images out of the box, but it also can’t even learn to do it

**Comment by geroldmeisinger** [comment]
my controlnet model here can guide light and dark areas https://civitai.com/models/137638?modelVersionId=157808
